### PR TITLE
[res/TensorFlowPythonExamples] Add unique operation case

### PR DESCRIPTION
**Pull request by dr-venkman**
Parent Issue : #2568
Draft PR : #2829 

This Commit will add Unique Operation related case.

ONE-DCO-1.0-Signed-off-by: venkat.iyer <venkat.iyer@samsung.com>

### CODE REVIEW

File: res/TensorFlowPythonExamples/examples/unique/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import tensorflow as tf
+
+in_ = tf.compat.v1.placeholder(dtype=tf.float32, shape=(1, 6), name="unique_in")
+op_ = tf.compat.v1.math.unique(in_)
+
```
Comment (seanshpark) [inline res/TensorFlowPythonExamples/examples/unique/__init__.py:5]: please remove this additional line

File: res/TensorFlowPythonExamples/examples/unique/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import tensorflow as tf
+
+in_ = tf.compat.v1.placeholder(dtype=tf.float32, shape=(1, 6), name="unique_in")
```
Comment (seanshpark) [inline res/TensorFlowPythonExamples/examples/unique/__init__.py:3]: other examples use input name `Hole`. please use `Hole`

File: res/TensorFlowPythonExamples/examples/unique/__init__.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+op_ = tf.compat.v1.math.unique(in_)
```
Comment (seanshpark) [inline res/TensorFlowPythonExamples/examples/unique/__init__.py:4]: this has error
```
AttributeError: module 'tensorflow._api.v1.compat.v1.math' has no attribute 'unique'
```

**Comment by seanshpark** [comment]
have to tested this script?

File: res/TensorFlowPythonExamples/examples/unique/__init__.py (hunk 1 above)
Comment (dr-venkman) [reply res/TensorFlowPythonExamples/examples/unique/__init__.py]: Removed, thank you.

File: res/TensorFlowPythonExamples/examples/unique/__init__.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+import tensorflow as tf
+
+in_ = tf.compat.v1.placeholder(dtype=tf.float32, shape=(1, 6), name="Hole")
+op_ = tf.compat.v1.unique(in_)
```
Comment (seanshpark) [inline res/TensorFlowPythonExamples/examples/unique/__init__.py:4]: you now need a new line at the end of this line

File: res/TensorFlowPythonExamples/examples/unique/__init__.py (hunk 2 above)
Comment (dr-venkman) [reply res/TensorFlowPythonExamples/examples/unique/__init__.py]: renamed to "Hole".

File: res/TensorFlowPythonExamples/examples/unique/__init__.py (hunk 3 above)
Comment (dr-venkman) [reply res/TensorFlowPythonExamples/examples/unique/__init__.py]: Yes, the correct api is `tf.compat.v1.unique`, I fixed the line. Thanks for spotting.

**Comment by seanshpark** [comment]
Error
```
ValueError: Shape must be rank 1 but is rank 2 for 'Unique' (op: 'Unique') with input shapes: [1,6].
```

**Comment by seanshpark** [comment]
Asking again. Have you tested this script?

**Comment by dr-venkman** [comment]
> Asking again. Have you tested this script?

Yes, I fixed it now. Thanks.

**Comment by dr-venkman** [comment]
> Asking again. Have you tested this script?

I fixed the shape error. Thanks for spotting.

File: res/TensorFlowPythonExamples/examples/unique/__init__.py (hunk 4 above)
Comment (dr-venkman) [reply res/TensorFlowPythonExamples/examples/unique/__init__.py]: Fixed, thanks.

**Pull request merged**